### ISSUE TITLE
Remove deprecated t.web.twcgi.PHPScripts

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5456]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5456|
|Type|task|
|Created|2012-01-21 21:00:18Z|
|Branch|https://github.com/twisted/twisted/tree/twcgi-php-5456|

in [twisted/web/twcgi.py](twisted/web/twcgi.py), `PHPScript` and `PHP3Script` are deprecated since 10.1 ([#7275](https://github.com/twisted/twisted/issues/7275), r28559) and can be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__5456 5456
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_twcgi_php_5456 branches/twcgi-php-5456
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1327179618000000 1327179618000000
changetime__1328548821000000 1328548821000000
version__None None
owner__thijs thijs
cc__thijs cc__jknight cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [33505]) Branching to 'twcgi-php-5456'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [33506]) removed deprecated FilteredScripts for php, refs [#5456](https://github.com/twisted/twisted/issues/5456)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for following up on all these old deprecations, thijs. Looks great, [build results are fine](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/twcgi-php-5456|), please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [33517]) Merge twcgi-php-5456: Removed deprecated `PHP3Script` and `PHPScript` from twisted.web.twcgi. Use `twcgi.FilteredScript` instead.

Author: thijs
Reviewer: jesstess
Fixes: [#5456](https://github.com/twisted/twisted/issues/5456)
